### PR TITLE
Fix filter UI added Property field

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for Filter UI update and added Property #572 

Before : 
Screenshot 2023-07-27 at 5 55 00 PM

After:
Screenshot 2023-07-27 at 5 55 28 PM

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 25 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

File: django_project/frontend/src/containers/MainPage/SideBar/Filter.tsx [hunk 1]
```diff
@@ -145,10 +137,55 @@ function Filter() {
...
     useEffect(() => {
-        fetchSpeciesList()
+        fetchSpeciesList();
+        fetchPropertyList();
     }, [])
 
+    const handleDeleteSpecies = (valueToDelete: string) => {
+        const updatedSelectedSpecies = selectedSpecies.filter((item) => item !== valueToDelete);
+        setSelectedSpecies(updatedSelectedSpecies);
+        dispatch(toggleSpecies(updatedSelectedSpecies.length > 0 ? updatedSelectedSpecies.join(',') : ''));
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/SideBar/Filter.tsx:161]: Please avoid repeating yourself multiple times. You can create a `useEffect` for `selectedSpecies` and then run `dispatch` within it.
```
useEffect(() => {
    dispatch(toggleSpecies(selectedSpecies.length > 0 ? selectedSpecies.join(',') : ''));
}, [selectedSpecies])
```

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 20 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 12 min

**Pull request merged**